guide/src/templates/ignoring.md

```diff
@@ -14,3 +14,6 @@ ignore = [
 ```
 
 Both files and folders may be ignored using this method, but currently wildcards are **not supported**.
+
+Note that `cargo-generate` checks for which files to ignore after the removal of any `.liquid` file extensions.
+Meaning; Setting `ignore` to `["file.txt"]` will result in the ignoring of a file named `file.txt.liquid`.
```

Comment:
I'd say "will result" could be clearer by saying "will also result". But it's just a technicality.